<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DH</class>
 <widget class="QDialog" name="DH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="outputTextEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>401</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>OutPut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateAllButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GenerateAll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputXBLineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>XB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputXALineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>XA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>P_digit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
